src/__tests__/utils/test-utils.tsx

```diff
@@ -17,17 +22,19 @@ export function render(ui: React.ReactElement, themeConfig?: ThemeConfig) {
     const { protocolTheme } = getProtocolTheme('light');
     const theme = themeConfig ? createTheme(themeConfig) : protocolTheme;
     return (
-      <ThemeProvider theme={theme}>
-        <SnackbarProvider
-          maxSnack={3}
-          autoHideDuration={6000}
-          action={(snackbarKey) => (
-            <SnackbarCloseButton snackbarKey={snackbarKey} />
-          )}
-        >
-          <Provider store={store}>{children}</Provider>
-        </SnackbarProvider>
-      </ThemeProvider>
+      <StyledEngineProvider injectFirst>
```

Comment:
I added it in the tests for consistency, but it doesn't look like the snapshots look at the styles anyway...